### PR TITLE
{bio}[foss/2022a] topaz v0.2.5.230926 w/ CUDA 11.7.0 w/ PyTorch 1.12.0

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
i'm ok with the name `topaz-em`, but we already have easyconfigs for this software called `topaz`, so that's a bit unfortunate

File: easybuild/easyconfigs/t/topaz-em/topaz-em-0.2.5.20230926-foss-2022a-CUDA-11.7.0.eb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+description = """Particle picking software for single particle cryo-electron microscopy using    
+convolutional neural networks and positive-unlabeled learning. Includes methods
+for micrograph denoising."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+use_pip = True
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
```
[the remark this answers is not in the document]
Comment (ThomasHoffmann77) [reply easybuild/easyconfigs/t/topaz-em/topaz-em-0.2.5.20230926-foss-2022a-CUDA-11.7.0.eb:22]: > can't we use `pkgconf` here?

I'll give it a try

**Comment by ThomasHoffmann77** [comment]
> i'm ok with the name `topaz-em`, but we already have easyconfigs for this software called `topaz`, so that's a bit unfortunate

I'll rename it to topaz, as this is the name of the software and the name of the github repository.

**Comment by smoors** [comment]
> @smoors Is it ok to receive the latest bugfix patch by github.com/account/repo/compare/version..commit.patch?

hm, that's creative :)
the patch is quite big to be included in the easyconfigs repo though.
why not use the commit tarball directly?

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**